pkgs/development/libraries/science/biology/elastix/default.nix

```
@@ -1,13 +1,14 @@
-{ stdenv, fetchurl, cmake, itk, python }:
+{ stdenv, fetchFromGitHub, cmake, itk, python }:
 
 stdenv.mkDerivation rec {
   pname    = "elastix";
-  pversion = "4.9.0";
-  name  = "${pname}-${pversion}";
+  version = "5.0.0";
 
-  src = fetchurl {
-    url    = "https://github.com/SuperElastix/${pname}/archive/${pversion}.tar.gz";
-    sha256 = "02pbln36nq98xxfyqwlxg7b6gmigdq4fgfqr9mym1qn58aj04shg";
+  src = fetchFromGitHub {
+    owner  = "SuperElastix";
+    repo   = pname;
+    rev    = "4c1441bcbc29782b5cbf100a40a433e6689d5ffe";
```

```suggestion
    rev    = version;
```
to fetch upstream tag named after the declared version.

It makes the link between source and version clearer, plus it allows auto-updates of the pkg by nixpkgs-update.

---

Done.